master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
If stats_service has nothing particular, I would prefer not particularize it, and put it in the genereic services bucket.

c['services'].append(StatsService(config, for, stats))